components/faculty-card.tsx

```diff
@@ -0,0 +1,64 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { MdEmail } from 'react-icons/md';
+import { MdPhone } from 'react-icons/md';
```

Comment:
This should merge into a single import.
```suggestion
import { MdEmail, MdPhone } from 'react-icons/md';
```